packages/nextjs/app/builders/0x119d9A1ef0D16361284a9661727b363B04B5B0c8/page.tsx

```diff
@@ -0,0 +1,178 @@
+"use client";
+
+import Image from "next/image";
+import { GitHubIcon, LinkedInIcon, TwitterIcon } from "./icons";
+import type { NextPage } from "next";
+import { Address } from "~~/components/scaffold-eth";
+
+const DmysticalCoderProfile: NextPage = () => {
+  const myAddress = "0x119d9A1ef0D16361284a9661727b363B04B5B0c8";
+
+  // Define skills in arrays for easier management
+  const blockchainSkills = ["Solidity", "Foundry", "Hardhat", "Ethers"];
+  const frontendSkills = ["React", "TypeScript", "NextJS", "TailwindCSS"];
```

Comment:
Since these are all constant, it makes sense to place them outside of the component. So they're created only once at module initialization, rather than every render.